src/Workspaces/Core/MSBuild.BuildHost/BuildHost.cs

```diff
@@ -14,23 +14,25 @@
 using Microsoft.Build.Logging;
```

Comment:
> taking the dependency (even though
it's open source) would produce a long-term maintenance burden.

this seems crazy to me.  Where is the issue here?  VS is betting big on streamjsonrpc.  We already use it for OOP heavily.  It's the foundation of gladstone.  What part of our org is making it challenging for a first party like us to use this?

Reply:
Discussed offline.